src/utility/Helpers.tsx

```diff
@@ -6,7 +6,7 @@
  */
 export const NewlineText = (text: string) => {
   const oldText = text;
-  const newText = oldText.split("\n").map((str) => <p>{str}</p>);
+  const newText = oldText.split("\n").map((str, index) => <p key={index}>{str}</p>);
```

Comment:
Can you explain why you added this change?

Reply:
to fix the key error that was showing in the console